libs9s/s9srpcclient.cpp

```diff
@@ -10420,6 +10421,13 @@ S9sRpcClient::getDbVersions(
     request["vendor"] = vendor;
     if(patchNumber)
         request["patch_number"] = true;
+    if(useVendorApi)
+    {
+        if(patchNumber)
```

Comment:
same `if` in line 10422, is it possible to have to avoid this `if` duplication?

Reply:
Not sure if it makes sense to have only one `if(patchNumber)` in this part of code...